new_lessons/sets.md

```diff
@@ -0,0 +1,337 @@
+# Sets 
+
+## Introduction 
+
+Sets are a unique data structure in Python.  Sets can be used to store data like lists and dictionaries, but sets also have a wide array of built in functionality based on the concept of sets in mathematics, and this is where sets really shine!  In this lesson we will explore the basics of sets and look at the ways we can take advantage of the features of sets in algorithms and problem solving.
+
+## Vocabulary and Synonyms 
+
+| Vocab | Definition | Synonyms | How to Use in a Sentence |
+| ----------------- | ----------------------------------------------------------------------------------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Set |  |  |
+| Union |  |  |
+| Intersection |  |  |
+| Difference |  |  |
+| Disjoint |  |  |
+
+## Set Basics
```

Comment:
As mentioned in another comment, if there's another word for "Basics," such as "Fundamentals," or only "Sets," I think that'd be great.